test/codeu/chat/server/JSONTest.java

```diff
@@ -187,10 +231,36 @@ public void orderDoesntMatter() {
         JSON log = new JSON();
         Model model = new Model();
         Model expected = new Model();
-        expected.add(new User(createUuid("1.1074501217"), "Krager", Time.fromMs(4098590)));
-        expected.add(new User(createUuid("1.3566231147"), "FakeUserName1234", Time.fromMs(48574)));
-        expected.add(new ConversationHeader(createUuid("1.3959833157"), createUuid("1.1074501217"),  Time.fromMs(98094), "Chat1"), new ConversationPayload( createUuid("1.3959833157"), createUuid("1.3350517446"), createUuid("1.1516759239")));
-        expected.add(new ConversationHeader(createUuid("1.3959833157"), createUuid("1.1074501217"),  Time.fromMs(59485), "Chat2"), new ConversationPayload( createUuid("1.3959833157"), createUuid("1.3350517446"), createUuid("1.1516759239")));
+        User user1 = new User(createUuid("1.1074501217"), "Krager", Time.fromMs(4098590));
```

Comment:
Is this the exact same code as above? If so, you might think about putting it in a helper function or an initializer or something.